jquery.fitvids.js

```diff
@@ -9,7 +9,20 @@
 *
 */
 
-;(function( $ ){
+!function(factory) {
+  if ("object" == typeof exports && "undefined" != typeof module) {
+    module.exports = factory;
+  }
+  else if ("function" == typeof define && define.amd) {
+    define(['jquery'], factory);
+  }
+  else {
+    var n;
+    "undefined" != typeof window ? n = window : "undefined" != typeof global ? n = global : "undefined" != typeof self && (n = self);
```

Comment:
this line takes a while to parse for humans. There's no deal with trying to make code as small as possible in non-minified source so let's format it.